packages/lambda-tiler/src/routes/attribution.ts

```diff
@@ -92,6 +92,7 @@ async function tileSetAttribution(
   const host = await config.Provider.get(config.Provider.id('linz'));
 
   for (const layer of filteredLayers) {
+    if (layer.disabled) continue;
```

Comment:
When we make individual tile sets do we turn disabled off?

Reply:
I think individual TileSet are created with hard coded values(minzoom:0, maxzoom:32) and without disable flag in it.